tests/plugins/test_resolve_composes.py

```diff
@@ -254,7 +254,7 @@ def test_request_compose(self, workflow, reactor_config_map):
         self.run_plugin_with_args(workflow, reactor_config_map=reactor_config_map)
 
     @pytest.mark.parametrize('arches', (
-        ['x86_64', 'ppc64le'],
+        ['ppc64le', 'x86_64'],
```

Comment:
Was this change needed here for some reason or is it just a consequence of the 'parametrize' in line 273 for `test_request_compose_for_modules`?

Reply:
test wasn't working as intended, to define arches for resolve_compose, arches has to be in check_and_set_platforms plugin, ALSO odcsclient is mocked also in workflow, so it was basically ignoring our mock with arches, I am adding here once() so it is more clearer

Reply:
Thanks for the reply